cluster/addons/dns/kube2sky/kube2sky.go

```diff
@@ -222,6 +226,58 @@ func (ks *kube2sky) handleEndpointAdd(obj interface{}) {
 	}
 }
 
+func (ks *kube2sky) handlePodCreate(obj interface{}) {
+	if e, ok := obj.(*kapi.Pod); ok {
+		//If the pod ip is not yet available, do not attemt to create
+		if e.Status.PodIP != "" {
```

Comment:
if this happens, will the eventual setting of podIP trigger a watch update?  I don't know if status will trigger it.

@lavalamp